eng/Build.props

```diff
@@ -117,6 +120,17 @@
         <ProjectToBuild Condition=" '$(BuildJava)' == 'true'" Include="@(JavaProjects)" Exclude="@(ProjectToExclude)" />
         <ProjectToExclude Condition=" '$(BuildJava)' != 'true'" Include="@(JavaProjects)" />
 
+        <!-- These projects have a transitive dependency on Microsoft.AspNetCore.Components.Web.JS exclude these projects if we are not building NodeJS -->
```

Comment:
- Is this two sentences? I find it a bit hard to parse.
- Minor but please include periods in your comments.